README.md

```diff
@@ -1,33 +1,26 @@
-# parodos-dev.github.io webpage
-
-## Build
-
-```bash
-make build
-```
-
-## Check links
-
-```bash
-make check-link
-```
-
-## Run site locally
-
-```bash
-make run
-```
-
-The site is available at `http://localhost:4000/`
-
-## Contrib
-
-Template based on `github.com/themefisher/kross-jekyll` project.
-
-## Documentation version update
-
-On documentation version change, the following changes need to be done:
-
-- [ ] `cp _data/documentation/latest.yaml _data/documentation/$VERSION.yaml`
-- [ ] `cp -rfv /documentation/latest /documentation/$VERSION`
-- [ ] edit /documentation/$VERSION/operations/api.html and change the swagger URL
+parodos-dev.github.io 
+
+This site is built by hugo static site generater and published using a github action to https://parodos.dev
+
+# Devleopment
+- Requiremens
+    - go
+    - git
+    - Download `hugo` *exetended* verision from [hugo releases page](https://github.com/gohugoio/hugo/releases/)
+
+- Run it 
+    ```bash
+    hugo server
+    ```
+
+# Content Organization
+- content/docs
+    The main directory for the project document 
+- content/docs/workflows
+    Document for the selected set of workflows, for https://parodos-dev/serverless-workflows
+- content/docs/workflows-examples
+    Document for the examples workflows, for https://parodos-dev/serverless-workflow-examples
+- content/post
+    Articles, blog posts, etc.
```

Comment:
can you add a new line at the end of the "header" in the list? the paragraph is rendered in line

Reply:
done